templates/gunicorn.py.j2

```diff
@@ -0,0 +1,12 @@
+bind = '{{ gunicorn_bind }}'
+workers = {{ gunicorn_workers }}
+accesslog = '-'
+errorlog = '{{ gunicorn_log }}'
+
+{% if develop or test -%}
```

Comment:
I think `reload` and `loglevel` should be broken out into their own variables. Defaults can be whatever Gunicorn defaults to if they aren't set.